ios/FluentUI/Pill Button Bar/PillButtonBar.swift

```diff
@@ -115,13 +115,15 @@ open class PillButtonBar: UIScrollView, FluentUIWindowProvider {
         }
     }
 
-    private var stackView = UIStackView()
+    private lazy var stackView: UIStackView = {
+        initStackView()
+    }()
```

Comment:
Looks good! And I think it can be simplified even further. The closure here simply invokes this method when run, so it can be optimized away like so:

```swift
private lazy var stackView: UIStackView = initStackView()
```

Reply:
Same with `unreadDotLayer` above.